README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
-* Install [IntelliJ IDEA](https://www.jetbrains.com/idea/) optionally for creating/running unit tests.
-* Ensure you have a valid Elastic Path development environment.
-
-### Configuration
-##### `ep.config.json (options)`
-
- - `cortexApi.path`, **required**, *string*:
-The URL containing the hostname and port for the storefront to use to reach Cortex. For your convenience, a web proxy is configured in this project's Webpack configuration. Set the value to `/cortex` for local development to redirect Cortex calls to the local proxy.
- - `cortexApi.scope`, **required**, *string*:
-Name of store to retreive data from Cortex.
- - `cortexApi.pathForProxy`, **required**, *string*:=
- The path the webpack proxy will route storefront Cortex calls to. URL consisting of hostname and port to actual running instance of Cortex. Leave this value empty to disable the proxy.
- - `skuImagesUrl`, **required**, *string*:
- Path to catalog images hosted on an external CMS. Set this to the full URL of your images, replacing the sku/file-name with the string `%sku%`. This value will be populated during pageload with values retreived by Cortex.
-  - `siteImagesUrl`, **optional**, *string*:
- Path to site content and marketing images hosted on an external CMS. Set this to the full URL of your images, replacing the file-name and file-extension with the string `%fileName%`. This value will be populated during pageload with values set in your components and use assets locally available in `./src/images/site-images` as the fallback.
- - `enableOfflineMode`, **optional**, *bool*:
-  The option to enable offline mode. During offline mode, requests are fetched from static data instead of Cortex. For more information on offline mode, see [how it works](#offline-mode).
- - `gaTrackingId`, **optional**, *string*:
- The Google Analytics tracking ID to integrate with Google Analytics Suite. Track enhanced ecommerce activity as it happens on your site.
-
-### Sample data:
-The project includes a set of sample data for your convenience. **Note:** You must have a valid Elastic Path development environment prior to using the sample data.<br/>
-1. Extract the sample catalog data contents from this project `ep-store/data` into your `ep-commerce/extensions/database/ext-data/src/main/resources/data` directory.
-2. Update `ep-commerce/extensions/database/ext-data/src/main/resources/data/liquibase-changelog.xml`.<br/>
-Add the following sample data to the file. **Note:** It should be the only sample data included within the sample data blocks: `<include file="ep-blueprint-data/liquibase-changelog.xml" relativeToChangelogFile="true" />`
-3. Update `ep-commerce/extensions/database/ext-data/src/main/resources/environments/local/data-population.properties`. Set the liquibase.contexts property to `default,ep-blueprint`.
+* [Java JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html)
```

Comment:
I'd prefer us keep the mention that "Java, Maven, and IntelliJ" are optional and only required for running UT. Reason being I can see someone picking up this asset and saying "wait, I need Java and Maven just to run this?!"